src/viadot/utils.py

```diff
@@ -835,3 +834,92 @@ def skip_test_on_missing_extra(source_name: str, extra: str) -> None:
         msg,
         allow_module_level=True,
     )
+
+
+def anonymize_df(
+    df: pd.DataFrame,
+    columns: list[str],
+    method: Literal["mask", "hash"] = "mask",
+    value: str = "***",
+    date_column: str | None = None,
+    days: int | None = None,
```

Comment:
No idea.

Reply:
Can you rename this to `older_than_days` pls? I just had too read the code closely to understand what this param actually means because `days` is so generic it might as well be `x`, so I found a better name for it.

Reply:
since the logic works fine and this change would require extra modifications in connectors, let's move it to the backlog - issue https://github.com/dyvenia/viadot/issues/1058